bootstrap/osx_bootstrap.sh

```diff
@@ -25,9 +25,9 @@ fi
 echo -e "\n${GREEN} --->  Installing Tesseract ${NC} \n"
 if command -v tesseract -v >/dev/null 2>&1; then
     echo -e "\n${GREEN}  --->  Skipping Tesseract install. Already installed. ${NC}\n"
-    brew upgrade tesseract --with-all-languages --with-training-tools --with-serial-num-pack
```

Comment:
`--with-all-languages --with-training-tools --with-serial-num-pack` no longer exists in Brew